docs/pilot.md

```diff
@@ -107,78 +110,75 @@ Any feedback is welcome, and questions or problems reports are welcome as well,
+   LittleCMS/2.9-GCCcore-9.3.0                     cURL/7.69.1-GCCcore-9.3.0                            re2c/1.3-GCCcore-9.3.0
+   METIS/5.1.0-GCCcore-9.3.0                       cairo/1.16.0-GCCcore-9.3.0                           scikit-build/0.10.0-foss-2020a-Python-3.8.2
+   MPFR/4.0.2-GCCcore-9.3.0                        double-conversion/3.1.5-GCCcore-9.3.0                snappy/1.1.8-GCCcore-9.3.0
+   Mako/1.1.2-GCCcore-9.3.0                        expat/2.2.9-GCCcore-9.3.0                            util-linux/2.35-GCCcore-9.3.0
+   MariaDB-connector-c/3.1.7-GCCcore-9.3.0         flex/2.6.4-GCCcore-9.3.0                             x264/20191217-GCCcore-9.3.0
+   Mesa/20.0.2-GCCcore-9.3.0                       fontconfig/2.13.92-GCCcore-9.3.0                     x265/3.3-GCCcore-9.3.0
+   Meson/0.55.1-GCCcore-9.3.0-Python-3.8.2         foss/2020a                                           xorg-macros/1.19.2-GCCcore-9.3.0
+   NASM/2.14.02-GCCcore-9.3.0                      freetype/2.10.1-GCCcore-9.3.0                        zstd/1.4.4-GCCcore-9.3.0
+   NLopt/2.6.1-GCCcore-9.3.0                       gompi/2020a
+   NSPR/4.25-GCCcore-9.3.0                         gperf/3.1-GCCcore-9.3.0
 ```
 
-### Build host
-
-* CentOS 7.8.2003
-* Intel Xeon CPU E5-2680 (`haswell`)
-* Singularity container with CentOS 7.8.2003 + `cvmfs` 2.7.3
-
 ### EasyBuild configuration
 
-The latest `develop` version of EasyBuild was used, all changes required to install the software in the `2020.08` version of the pilot repository will be included in the upcoming release of EasyBuild v4.3.0.
+The latest `develop` version of EasyBuild was used, all changes required to install the software in the `2020.09` version of the pilot repository will be included in the upcoming release of EasyBuild v4.3.1.
 
+An example configuration of the build environment based on https://github.com/EESSI/software-layer can be seen here:
 ```
 $ eb --show-config
 #
 # Current EasyBuild configuration
 # (C: command line argument, D: default value, E: environment variable, F: configuration file)
 #
-buildpath      (E) = /tmp/easybuild/build
-containerpath  (E) = /tmp/easybuild/containers
-debug          (E) = True
-ignore-osdeps  (E) = True
-installpath    (E) = /cvmfs/pilot.eessi-hpc.org/2020.08/software/x86_64/intel/haswell
-packagepath    (E) = /tmp/easybuild/packages
-prefix         (E) = /tmp/easybuild
-repositorypath (E) = /tmp/easybuild/ebfiles_repo
-robot-paths    (D) = /home/eessi/easybuild-easyconfigs/easybuild/easyconfigs
-rpath          (E) = True
-sourcepath     (E) = /tmp/easybuild/sources
-sysroot        (E) = /cvmfs/pilot.eessi-hpc.org/2020.08/compat/x86_64
-trace          (E) = True
+buildpath         (D) = /tmp/terjekv/singularity-home/.local/easybuild/build
+containerpath     (D) = /tmp/terjekv/singularity-home/.local/easybuild/containers
```

Comment:
Please edit a bit to make it "anonymous" (no `terjekv`, like it was before)

Reply:
Good point! Will do.